Sources/SwiftProtobuf/Message.swift

```diff
@@ -76,18 +96,19 @@ public protocol Message: CustomDebugStringConvertible {
   /// then invokes `decodeField` above.
   mutating func decodeMessage<D: Decoder>(decoder: inout D) throws
```

Comment:
Scratch that, I think this was the one we wanted to keep public, but decodeField was the one we were sorta hopping to kill/making internal/etc.